contracts/token/ERC721/ERC721.sol

```diff
@@ -446,47 +432,14 @@ abstract contract ERC721 is Context, ERC165, IERC721, IERC721Metadata, IERC721Er
         }
     }
 
-    /**
-     * @dev Hook that is called before any token transfer. This includes minting and burning. If {ERC721Consecutive} is
-     * used, the hook may be called as part of a consecutive (batch) mint, as indicated by `batchSize` greater than 1.
-     *
-     * Calling conditions:
-     *
-     * - When `from` and `to` are both non-zero, ``from``'s tokens will be transferred to `to`.
-     * - When `from` is zero, the tokens will be minted for `to`.
-     * - When `to` is zero, ``from``'s tokens will be burned.
-     * - `from` and `to` are never both zero.
-     * - `batchSize` is non-zero.
-     *
-     * To learn more about hooks, head to xref:ROOT:extending-contracts.adoc#using-hooks[Using Hooks].
-     */
-    function _beforeTokenTransfer(address from, address to, uint256 firstTokenId, uint256 batchSize) internal virtual {}
-
-    /**
-     * @dev Hook that is called after any token transfer. This includes minting and burning. If {ERC721Consecutive} is
-     * used, the hook may be called as part of a consecutive (batch) mint, as indicated by `batchSize` greater than 1.
-     *
-     * Calling conditions:
-     *
-     * - When `from` and `to` are both non-zero, ``from``'s tokens were transferred to `to`.
-     * - When `from` is zero, the tokens were minted for `to`.
-     * - When `to` is zero, ``from``'s tokens were burned.
-     * - `from` and `to` are never both zero.
-     * - `batchSize` is non-zero.
-     *
-     * To learn more about hooks, head to xref:ROOT:extending-contracts.adoc#using-hooks[Using Hooks].
-     */
-    function _afterTokenTransfer(address from, address to, uint256 firstTokenId, uint256 batchSize) internal virtual {}
-
     /**
      * @dev Unsafe write access to the balances, used by extensions that "mint" tokens using an {ownerOf} override.
      *
      * WARNING: Anyone calling this MUST ensure that the balances remain consistent with the ownership. The invariant
      * being that for any address `a` the value returned by `balanceOf(a)` must be equal to the number of tokens such
      * that `ownerOf(tokenId)` is `a`.
      */
-    // solhint-disable-next-line func-name-mixedcase
-    function __unsafe_increaseBalance(address account, uint256 value) internal {
+    function _increaseBalance(address account, uint256 value) internal virtual {
```

Comment:
Thoughts on naming this `_updateBalance` to indicate the analogy with `_update`? In some cases like ERC20Votes contracts have to override both functions so it makes sense to me to tie them together conceptually.

@ernestognw I see you had renamed `_updateBalance` to `_increaseBalance` in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4419, what makes you prefer that one?

Reply:
I think in `_updateBalance`, one would expect the value to replace the new one (overwrite), whereas `_increaseBalance` makes it clear that value is an delta (increment). I'd stick with `_increaseBalance` for clarity